### ISSUE TITLE
Alacritty on macOS introduces weird bash bug

### ISSUE THREAD

**Issue by hyperknot** [alacritty/alacritty#8180]
### System

OS: macOS
Version: alacritty 0.13.2 (bb8ea18)

### Logs

There is two weird bugs with `brew install alacritty`.

1. It installs a bash completion which is not compatible with the macOS bash 3.2.
```
++++ . /opt/homebrew/etc/bash_completion.d/alacritty
+++++ complete -F _alacritty -o nosort -o bashdefault -o default alacritty
bash: complete: nosort: invalid option name
```

2. It somehow overrides my bash to use the old version, really really weird.
Details on this thread: https://github.com/orgs/Homebrew/discussions/5614

Summary is this super weird behaviour:
```
echo $BASH && echo $BASH_VERSION
/opt/homebrew/bin/bash
5.2.32(1)-release
```

My config has nothing related to this, only font, selection, colors, window, keyboard.

**Comment by kchibisov** [comment]
alacritty doesn't create any files, including the completions. So whatever you got installed is done by your package manager of choice.

We also don't touch `SHELL` in anyway, other than reading what you have in the system.

**Comment by hyperknot** [comment]
OK, the completion is a homebrew packaging issue then. But what about the shell?

This is an impossible situation, and only Alacritty is doing this:
```
echo $SHELL && echo $BASH && echo $BASH_VERSION
/opt/homebrew/bin/bash
/opt/homebrew/bin/bash
3.2.57(1)-release
```

Terminal.app
```
echo $SHELL && echo $BASH && echo $BASH_VERSION
/opt/homebrew/bin/bash
/opt/homebrew/bin/bash
5.2.32(1)-release
```

WezTerm:
```
echo $SHELL && echo $BASH && echo $BASH_VERSION
/opt/homebrew/bin/bash
/opt/homebrew/bin/bash
5.2.32(1)-release
```

iTerm:
```
echo $SHELL && echo $BASH && echo $BASH_VERSION
/opt/homebrew/bin/bash
/opt/homebrew/bin/bash
5.2.32(1)-release
```

**Comment by kchibisov** [comment]
I guess the `BASH_VERSION` is just wrong, but  as you can see, all the apps have the same path, and given that we don`t touch `BASH_VERSION`, maybe this variable is not updated at all or inherited?

We don't use bash even internally to launch anything, we use `zsh` for some path, but if you have `bash` set in the `alacritty.toml` it'll be launched anyway.

Maybe you've set the bash in the config by just `bash` and not by its full path. Try

```toml
shell = { program = "/opt/homebrew/bin/bash", args = [ "-l" ] }
```

**Comment by hyperknot** [comment]
No, it's not just a wrong string, the behaviour is also Bash 3.2, it's throwing the 
```
bash: complete: nosort: invalid option name
```
error, see above.

I just tried renaming my config to .bak and the behaviour is the same with a totally clean config.

Is this something which homebrew packaging could potentially do to Alacritty?

**Comment by hyperknot** [comment]
Yes, I can confirm that the website downloaded DMG is not producing this behaviour. So it's probably something with the linking in homebrew packaging which is ruining the app.

**Comment by hyperknot** [comment]
I opened a bug for homebrew: https://github.com/Homebrew/homebrew-cask/issues/185883
